package.json

```diff
@@ -1,20 +1,30 @@
 {
   "name": "@cloudflare/workers-types",
-  "version": "2.2.2",
+  "version": "3.0.0",
   "description": "TypeScript typings for Cloudflare Workers",
   "repository": {
     "type": "git",
     "url": "https://github.com/cloudflare/workers-types"
   },
   "main": "index.js",
```

Comment:
Looks like this file doesn't exist – and it wouldn't be uploaded either via the new `"files"` list addition.

```suggestion
```